cli/src/test/java/org/jboss/gm/cli/MainTest.java

```diff
@@ -70,6 +71,7 @@ public void reset() {
         // Reset the daemon between tests : https://discuss.gradle.org/t/stopping-gradle-daemon-via-tooling-api/16004/2
         // Under 4.10 the daemon appears to cache Config values which corrupt the tests.
         ConnectorServices.reset();
+        ConfigCache.getOrCreate(Configuration.class).reload();
```

Comment:
So both `reset` and `reload` are required? I can only go by the original comment that "the daemon appears to cache Config values which corrupt the tests".

Reply:
I believe we both investigated this before. I'm now seeing other spurious errors and forcing a reload made them go away.

Reply:
Experimenting with a `ConfigCache.clear` instead which might be more consistent.

Reply:
Updated.